frontend/src/features/game/game-room.tsx

```diff
@@ -19,6 +20,19 @@ export const GameRoom: React.FC = () => {
 
   const isCurrentPlayerTurn = currentPlayerId === player?.id
 
+  const { sendMessage } = useWSContext()
+  const roomId = useAppSelector((state) => state.roomId)
+  const totalDrawCardsRemaining = useAppSelector(
+    (state) => state.currentGame?.totalDrawCardsRemaining,
+  )
+
+  if (!totalDrawCardsRemaining) {
```

Comment:
should we make it so that only the admin sends this message? otherwise we might get a bajillion game ended messages

Reply:
tru tru tru